docs/gaps/gaps.tex

```diff
@@ -0,0 +1,286 @@
+that compete for the fissile materials (ie. \gls{MOX} fuel fab competes with \gls{SFR} fuel fab for separated plutonium).
+In this case, if both fabrication plants have infinite buffer/throughput, one fabrication takes all the
+separated fissile material, leaving the other plant idle.
+
+\Cref{fig:mox} and \cref{fig:sfr} plot the fuel demand of \gls{MOX} \glspl{LWR} and \glspl{SFR},
+respectively, in an EG29 scenario.
+
+
+\begin{figure}[htbp!]
+	\begin{center}
+		\includegraphics[scale=0.6]{./images/mox_fuel.png}
+	\end{center}
+	\caption{\gls{MOX} fuel demand for EG29}
+	\label{fig:mox}
+\end{figure}
+\begin{figure}[htbp!]
+	\begin{center}
+		\includegraphics[scale=0.6]{./images/sfr_fuel.png}
+	\end{center}
+	\caption{\gls{SFR} fuel demand for EG29}
+	\label{fig:sfr}
+\end{figure}
+\FloatBarrier
+
+
+\subsection{Static Parameters}
+Various parameters, most notably throughput and capacity values of fuel cycle support facilities,
+need to adjust according to demand. With transition scenarios with increasing power demand, the 
+demand for fuel and fissile material generally increases over time. The current workaround is to either
+have a facility with infinite capacity, or to set the capacity to a manually-calculated (look-back method)
+maximum fuel demand. The infinite capacity method fails for EG29 and EG30, due to the greedy exchange
+model. The look-back method allows a complete simulation, but inefficiencies occur with the distribution
+of fissile material, since the facility with the higher demand has to fill up first in order for the facility
+with the lower demand to receive fissile material. 
+
+A fix for this is to have a dynamic throughput that adjusts according
+to predicted fuel demand, or, more realistically, have an archetype with fixed, reasonable, throughput/buffer
+that is deployed when there is fuel demand beyond its predicted throughput.
+
+\subsection{Individual Demand Estimation}
+Currently, the \Cyclus framework's market is completely agent-based. At every timestep, each agent `submits' its
+demand and supply and the trade is made. However, this individual frame can cause inefficiencies mentioned previously.
+A more efficient process would be to set all demands proportional to the fuel demand. An example is illustrated
+in \cref{diag:hd}.
+
+% Define block styles
+\tikzstyle{decision} = [diamond, draw, fill=blue!20, 
+text width=4.5em, text badly centered, node distance=3cm, inner sep=0pt]
+\tikzstyle{block} = [rectangle, draw, fill=blue!20, 
+text width=5em, text centered, rounded corners, minimum height=4em]
+\tikzstyle{line} = [draw, -latex']
+\tikzstyle{cloud} = [draw, ellipse,fill=red!20, node distance=3cm,
+minimum height=2em]
+
+
+\begin{figure}
+	\centering
+	\scalebox{0.7}{
+		\begin{tikzpicture}[align=center, node distance = 3cm and 3cm, auto]
+		% Place nodes
+		\node [block] (rea) {\texttt{Reactor}};
+		\node [cloud, below of=rea] (d1) {1. Demand: \mathbf{Fuel}};
```

Comment:
mathbf only allowed in math mode. This causes the build to crash on both of my computers.